act-framework.bs

```diff
@@ -138,8 +138,86 @@ Editor note: This section describes the required properties of the data output b
 Rule Aggregation {#output-aggregation}
```

Comment:
We need to define how aggregation should be done, in order to do benchmarking correctly. In benchmarking, we want to take results from accessibility experts, and compare them to results from ATTs. The problem is that experts don't report with the same level of detail that ATTs do. To compare data from one with the other, they need to be at the same level of specificity. If an expert tester reports pass/fail on a criterion level, per page, we'll need to do the same with the results from an ATT before we can compare, hence aggregation.